docs/src/modules/utils/getDemoConfig.js

```diff
@@ -81,18 +81,30 @@ function getLanguageConfig(demoData) {
 
 export default function getDemo(demoData) {
   const baseConfig = {
-    title: 'Material demo',
+    title: demoData.title,
     description: demoData.githubLocation,
     files: {
-      'index.html': `
-<body>
-  <!-- Fonts to support Material Design -->
-  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />
-  <!-- Icons to support Material Design -->
-  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
-  <div id="root"></div>
-</body>
-      `,
+      'public/index.html': `
+<!DOCTYPE html>
+<html lang="${demoData.language}">
```

Comment:
Well, this assumes a lot, but I guess it could be fine. We could also not mention the lang 🤔

Reply:
> Well, this assumes a lot, but I guess it could be fine. We could also not mention the lang thinking

It assumes nothing. This is the language that is used when creating the codesandbox.

Reply:
I was considering the use cases where:

- the developer might not translate the content of the demos from English to the locale they use to browse the docs.
- the developer would export the code sandbox into a CRA project and build on top of it, not necessarily build for the language he has.

On Ant Design even if you have the Chinese version of the documentation, the created templates on StackBlitz, CodePen, CodePen with `lang="en"`. 

No strong point of view. I think that it depends on the perspective. Happy with the current version.

Reply:
> the developer would export the code sandbox into a CRA project and build on top of it, not necessarily build for the language he has.

This use case wasn't even supported previously. Also see https://github.com/mui-org/material-ui/issues/22112 about a similar issue: reproducing the docs demo.

> On Ant Design even if you have the Chinese version of the documentation, the created templates on StackBlitz, CodePen, CodePen with lang="en".

And what does this tell us? I'm getting really frustrated with this point. They might not have considered it. They might have. They might already have a plan to fix it. They might consider it intended. Either way: The existence of something does not tell you anything about whether that thing is "good" or "bad". And it definitely does not tell you why this thing exist. It also is purely a point to deconstruct proposals. It might be somewhat helpful if you looked at all! similar implementations. But if you only tell me about one (assuming a genuine discussions where you don't omit the cases that support my proposal) I can't find any value in it.

I am not a code monkey that perpetuates the status quo. I want to improve existing solutions and fix existing problems. Neither of these goals can be achieved by copying existing things.

Reply:
I agree with all your points. You can ignore my comment. I have no real insight on what should be best to do with the `lang`.